### DISCUSSION TITLE
Basic Drawer

### DISCUSSION THREAD

**Discussion by ekul1988** [ant-design/ant-design#50941] [Q&A] [answered]
Hello, Is there a way to move the X to the right side of the drawer? It currently shows up on the left side  on default. 

![basicDrawer](https://github.com/user-attachments/assets/62e129b2-4f30-48cc-ba57-e4de238eae6e)

`const page = () => {
    const [open, setOpen] = useState(false);
    const [placement, setPlacement] = useState('left');
    const showDrawer = () => {
      setOpen(true);
    };
    const onClose = () => {
      setOpen(false);
    };
    const onChange = (e) => {
      setPlacement(e.target.value);
    };
    return (
      <>
        <Space>
          <Radio.Group value={placement} onChange={onChange}>
            <Radio value="top">top</Radio>
            <Radio value="right">right</Radio>
            <Radio value="bottom">bottom</Radio>
            <Radio value="left">left</Radio>
          </Radio.Group>
          <Button type="primary" onClick={showDrawer}>
            Open
          </Button>
        </Space>
        <Drawer
          title="Basic Drawer"
          placement={placement}
          closable={true}
          onClose={onClose}
          open={open}
          key={placement}
        >
          <p>Some contents...</p>
          <p>Some contents...</p>
          <p>Some contents...</p>
        </Drawer>
      </>
    );
  };`

**Comment by kiner-tang** [comment, marked as answer]
Maybe you can try to use `extra` instead of `closeIcon`.
https://stackblitz.com/edit/react-9yw2tw?file=demo.tsx
<img width="1392" alt="image" src="https://github.com/user-attachments/assets/269125f3-9bfe-40bd-80dc-d64c631fd2b5">
